Source/WebCore/rendering/GridMasonryLayout.h

```diff
@@ -41,8 +41,14 @@ class GridMasonryLayout {
     {
     }
 
+    enum class OverrideLogicalWidth {
+        DoNotOverride,
```

Comment:
We could also just have the two `MinContent`/`MaxContent` values and pass in the enum as an optional but I'm not sure which makes more sense for this case or is better. Something to think about.

Reply:
Sure, we could do something like that. Let me update this.